src/components/ChildFormFragment.tsx

```diff
@@ -0,0 +1,131 @@
+import { DatePicker, Form, Input, Radio, Upload } from 'antd';
+import { FormListFieldData } from 'antd/lib/form/FormList';
+import React from 'react';
+const { Dragger } = Upload;
+const { TextArea } = Input;
+
+const ChildFormFragment: React.FC<{ field: FormListFieldData }> = ({
+  field,
+}) => {
+  return (
+    <>
+      <Form.Item
+        name={[field.name, 'firstName']}
+        fieldKey={[field.fieldKey, 'firstName']}
+        label="First Name"
+        className="inline-block-half"
```

Comment:
Why class names? These dont make sense

Reply:
I agree, this does not make sense

Reply:
Also, is there a way to get the asterisk on the right side of the label? I think that is more standard. Not sure if it is simple enough with AntD forms...

Reply:
Who is this a question for haha